streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -21,30 +21,18 @@ import java.util
 import java.util.{Locale, Properties}
 import java.util.regex.Pattern
 import org.apache.kafka.common.serialization.{Serdes => SerdesJ}
-import org.apache.kafka.streams.kstream.{
-  Aggregator,
-  Initializer,
-  JoinWindows,
-  KGroupedStream => KGroupedStreamJ,
-  KStream => KStreamJ,
-  KTable => KTableJ,
-  KeyValueMapper,
-  Materialized => MaterializedJ,
-  Reducer,
-  StreamJoined => StreamJoinedJ,
-  Transformer,
-  ValueJoiner,
-  ValueMapper
-}
-import org.apache.kafka.streams.processor.{api, ProcessorContext}
+import org.apache.kafka.streams.kstream.{Aggregator, Initializer, JoinWindows, KeyValueMapper, Reducer, Transformer, ValueJoiner, ValueMapper, KGroupedStream => KGroupedStreamJ, KStream => KStreamJ, KTable => KTableJ, Materialized => MaterializedJ, StreamJoined => StreamJoinedJ}
+import org.apache.kafka.streams.processor.{ProcessorContext, api}
 import org.apache.kafka.streams.processor.api.{Processor, ProcessorSupplier}
 import org.apache.kafka.streams.scala.ImplicitConversions._
 import org.apache.kafka.streams.scala.serialization.{Serdes => NewSerdes}
 import org.apache.kafka.streams.scala.serialization.Serdes._
 import org.apache.kafka.streams.scala.kstream._
-import org.apache.kafka.streams.{KeyValue, StreamsBuilder => StreamsBuilderJ, StreamsConfig, TopologyDescription}
+import org.apache.kafka.streams.{KeyValue, StreamsConfig, TopologyDescription, StreamsBuilder => StreamsBuilderJ}
```

Comment:
Could you please revert those unrelated changes?

Reply:
Sorry for causing you trouble. 
Already revert